examples/runners/ash/src/main.rs

```diff
@@ -171,10 +171,13 @@ pub fn main() {
 }
 
 pub fn compile_shaders() -> Vec<SpvFile> {
-    let spv_paths: Vec<PathBuf> = vec![SpirvBuilder::new("examples/shaders/sky-shader")
-        .print_metadata(false)
-        .build()
-        .unwrap()];
+    let spv_paths: Vec<PathBuf> =
+        vec![
+            SpirvBuilder::new("examples/shaders/sky-shader", "spirv-unknown-vulkan1.1")
```

Comment:
```suggestion
            SpirvBuilder::new("examples/shaders/sky-shader", "spirv-unknown-vulkan1.0")
```

Reply:
The ash runner uses vulkan1.1, so this should stay vulkan 1.1. https://github.com/EmbarkStudios/rust-gpu/blob/8c89b49b6639fed7e8ef3c13b5c17c2b472ae680/examples/runners/ash/src/main.rs#L255